Realm/RLMAccessor.mm

```diff
@@ -29,6 +29,23 @@
 
 #import <objc/runtime.h>
 
+typedef NS_ENUM(char, RLMAccessorCode) {
+    RLMAccessorCodeChar,
```

Comment:
We do: an Int8 is actually a char (fixed length types only exist in the eyes of the compiler)

-Samuel E. Giddins

On Apr 20, 2015, at 9:21 AM, Ari Lazier <notifications@github.com> wrote:

In Realm/RLMAccessor.mm:

> @@ -29,6 +29,23 @@
>  
>  #import <objc/runtime.h>
>  
> +typedef NS_ENUM(char, RLMAccessorCode) {
> +    RLMAccessorCodeChar,
This naming is misleading since we don't support char in objc or swift.

—
Reply to this email directly or view it on GitHub.